modules/seqexec/server/src/main/scala/seqexec/server/gems/GemsController.scala

```diff
@@ -135,6 +146,27 @@ object GemsController {
                            useOI: OIUsage
   ) extends GemsConfig {
 
+    import Cwfs1Usage._
+    override val isCwfs1Used: Boolean = cwfs1 === UseCwfs1
+
+    import Cwfs2Usage._
+    override val isCwfs2Used: Boolean = cwfs2 === UseCwfs2
+
+    import Cwfs3Usage._
+    override val isCwfs3Used: Boolean = cwfs3 === UseCwfs3
+
+    import Odgw1Usage._
+    override val isOdgw1Used: Boolean = odgw1 === UseOdgw1
```

Comment:
Why not just `Odgw1Usage.UseOdgw1` here and elsewhere? Are you importing any other symbol from `Odgw1Usage`?

Reply:
... in which case why not a shorter name like `Odgw1Usage.Use`?

Reply:
About the first question, it is convenient that way because it is always used in `if-else` statements. The alternative would be to do write the comparison inside every `if`.
About the second, you are right. This is the code that motivated to ask in Slack about style, but I was to lazy to go back and fix that code. I'll change it.

Reply:
Sorry, what I meant was, why do `import FooUsage` and use `UseFoo` rather than skipping the import and saying `FooUsage.UseFoo`.

Reply:
Aaaahhh. Actually, I only care about the implicit `Eq`.